packages/create-jest/src/errors.ts

```diff
@@ -18,7 +18,7 @@ export class MalformedPackageJsonError extends Error {
   constructor(packageJsonPath: string) {
     super(
       `There is malformed json in ${packageJsonPath}\n` +
-        'Fix it, and then run "jest --init"',
+        'Fix it, and then run "create-jest" again',
```

Comment:
Good catch! I'm just not sure if the new suggestion is right. People using `--init` will see it too, right? Also user does not run `create-jest` directly. Hm.. Perhaps it's enough to say that the file is malformed?

Reply:
Sounds good, thanks for the idea!